src/scripts/clipperUI/animations/animationStrategy.ts

```diff
@@ -1,3 +1,5 @@
+import {SmartValue} from "../../communicator/smartValue";
```

Comment:
I am abusing the intended usage of `SmartValue`s here... I started using SVs because at one point, I thought I needed to be able to subscribe to changing `AnimationState`s.

Then it turned out I didn't need to subscribe, _but_ I still needed a value container to 1) persist ratings panel `AnimationState` whenever the ratings panel is re-rendered randomly w/o a state change (I don't know why this is happening, but it happens); and 2) update this persisted value across different layers of the animation strategy code (via `setAnimationStrategy`).

If someone has a suggestion for getting these behaviors without usings SVs (perhaps there's a more Mith-react way?), please let me know :) Thanks